cmd/add.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/mookjp/yakiire/lib"
+	"github.com/spf13/cobra"
+)
+
+var addCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add a document to a collection",
+	Long:  `Add a single document to a collection with a random ID`,
+	Run: func(cmd *cobra.Command, args []string) {
+		documentStr := args[0]
+		if documentStr == "" {
+			fmt.Printf("The document you entered seems to be empty!")
+			os.Exit(1)
+		}
+
+		var doc interface{}
+		err := json.Unmarshal([]byte(documentStr), &doc)
+		fmt.Println(doc)
+
+		flags := cmd.Flags()
+		collectionName, err := flags.GetString(cmdCollectionsKey)
+		if err != nil {
+			panic(err)
+		}
+
+		config := getConfig(cmd.Root())
+		cred := config.credentialPath
+		projectId := config.projectId
+
+		ctx := context.Background()
+		client, err := lib.NewClient(ctx, &lib.ClientConfig{
+			Credentials: cred,
+			ProjectID:   projectId,
+		})
+		if err != nil {
+			fmt.Printf("error: %+v", err)
+			os.Exit(1)
+		}
+
+		getCtx, _ := context.WithCancel(ctx)
+		res, err := client.Add(getCtx, collectionName, doc)
```

Comment:
Please use `Doc` and just print out JSON as a result so that the user can see the formated result with `jq` command.
See `README` how it should be worked with it.

Reply:
Ah I see. Done.